code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -61,13 +61,22 @@
 /obj/item/tank/jetpack/verb/toggle_rockets()
 	set name = "Toggle Jetpack Stabilization"
 	set category = "Object"
-	src.stabilization_on = !( src.stabilization_on )
-	to_chat(usr, "You toggle the stabilization [stabilization_on? "on":"off"].")
+
+	proc_toggle_rockets(usr)
+
+/obj/item/tank/jetpack/proc/proc_toggle_rockets(mob/user, var/list/message_mobs)
+	stabilization_on = !src.stabilization_on
+	to_chat(user, SPAN_NOTICE("You toggle \the [src]'s stabilization [stabilization_on ? "on" : "off"]."))
+	for(var/M in message_mobs)
+		to_chat(M, SPAN_NOTICE("[user] toggles \the [src]'s stabilization [stabilization_on ? "on" : "off"]."))
 
 /obj/item/tank/jetpack/verb/toggle()
 	set name = "Toggle Jetpack"
 	set category = "Object"
 
+	proc_toggle(usr)
+
+/obj/item/tank/jetpack/proc/proc_toggle(mob/user, var/list/message_mobs)
```

Comment:
This proc should call proc_toggle_rockets or whatever you rename it to, instead of doing the stabilization stuff itself.